### PR TITLE
[Ready] Toys and food. 

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

image

This will add more toys and food!

## Why It's Good For The Game

The new foods will make the chef work more fun!

The new toys are for fun, batong can be use by clowns or the crew to do silly jokes and pranks to sec or other crew members! the SM toy can be use for the same thing and use as a small grow tick!

SM toy can be only bought in a hacked enginner wardrobe.

## Changelog
:cl:
Add: Batong toy and SuperMatter toy.
add: Catburger , Sausagebread and egg with sausage.  
/:cl:

### CODE REVIEW

File: fulp_modules/things/toys/code/Vending_machine_code.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/obj/machinery/vending/games/Initialize()
```
Comment (CyoTheProtogen) [inline fulp_modules/things/toys/code/Vending_machine_code.dm:1]: why isn't this in fulp_modules >> container_init >> vending_init.dm?

Comment (JohnFulpWillard) [reply fulp_modules/things/toys/code/Vending_machine_code.dm:1]: BECAUSE IT DOESNT EXIST YET SWAIN MERGE MY PR

**Comment by EgorDinamit** [comment]
image

File: fulp_modules/container_init/vending_init.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/obj/machinery/vending/games/Initialize()
```
Comment (CyoTheProtogen) [inline fulp_modules/container_init/vending_init.dm:1]: The file extension is incorrect, meant to be .dm

File: tgstation.dme [hunk 3]
```diff
@@ -3328,12 +3334,10 @@
...
 #include "fulp_modules\mentorhelp\mentorwho.dm"
-#include "fulp_modules\jobs\job_integration.dm"
-#include "fulp_modules\jobs\brigdoc\code\brigdoc.dm"
-#include "fulp_modules\jobs\brigdoc\code\brigdoc_clothing.dm"
-#include "fulp_modules\jobs\brigdoc\code\gamemode_overwrite.dm"
-#include "fulp_modules\jobs\brigdoc\code\landmark.dm"
 #include "fulp_modules\mob_spawners\code\mobspawner.dm"
+#include "fulp_modules\things\food\code\food.dm"
+#include "fulp_modules\things\food\code\recipe.dm"
+#include "fulp_modules\things\toys\code\toys.dm"
```
Comment (SgtHunk) [inline tgstation.dme:3340]: Should probably remove the "things" folder altogether, and just have food and toys.

File: fulp_modules/things/food/code/food.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/obj/item/food/burger/catburger
+	name = "Catburger	"
```
Comment (SgtHunk) [inline fulp_modules/things/food/code/food.dm:2]:
```suggestion
	name = "catburger"
```

File: fulp_modules/things/food/code/recipe.dm [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+		/obj/item/food/bun = 1,
+		/obj/item/organ/ears/cat = 1,
+		/obj/item/organ/tail/cat = 1
+	)
+	result = /obj/item/food/burger/catburger
+	subcategory = CAT_BURGER
+
+
+/datum/crafting_recipe/food/meatbread
+	name = "Sausage bread "
```
Comment (SgtHunk) [inline fulp_modules/things/food/code/recipe.dm:13]:
```suggestion
	name = "Sausage bread"
```

File: fulp_modules/things/food/code/recipe.dm [hunk 6]
```diff
@@ -0,0 +1,19 @@
+/datum/crafting_recipe/food/catburger
+	name = "Cat person burger"
```
Comment (SgtHunk) [inline fulp_modules/things/food/code/recipe.dm:2]:
```suggestion
	name = "Catperson burger"
```

File: fulp_modules/things/toys/code/toys.dm [hunk 7]
```diff
@@ -0,0 +1,24 @@
+/obj/item/toy/plush/batong
+	name = "Batong"
```
Comment (SgtHunk) [inline fulp_modules/things/toys/code/toys.dm:2]:
```suggestion
	name = "batong"
```

**Review by SgtHunk — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 3 above)
Comment (DaddyIssues98) [reply tgstation.dme:3340]: yeah i was going to change the name of the folder lol

File: fulp_modules/container_init/vending_init.dm (hunk 2 above)
Comment (DaddyIssues98) [reply fulp_modules/container_init/vending_init.dm:1]: is  not .dm? weird, gonna fix

Comment (CyoTheProtogen) [reply fulp_modules/container_init/vending_init.dm:1]: Scratch that, issue on my end

**Comment by Helianthus00** [comment]
Fix the dme and I'm all for merging this

File: fulp_modules/container_init/vending_init.dm [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+    products += list(/obj/item/toy/plush/batong = 3)
```
Comment (JohnFulpWillard) [inline fulp_modules/container_init/vending_init.dm:2]: The digi shoes PR added the vending machine .dm file, please move these there

File: tgstation.dme [hunk 9]
```diff
@@ -3315,6 +3315,14 @@
 #include "fulp_modules\beefman_port\code\modules\mob\beefman.dm"
 #include "fulp_modules\beefman_port\code\modules\mob\beefman_clothing.dm"
 #include "fulp_modules\beefman_port\code\modules\mob\beefman_trauma.dm"
+#include "fulp_modules\container_init\vending_init.dm"
```
Comment (JohnFulpWillard) [inline tgstation.dme:3318]: Please make sure the .dme is good when you update it

**Pull request merged**